packages/react-effect/loom.config.ts

```diff
@@ -1,10 +1,9 @@
-import {createPackage, Runtime} from '@shopify/loom';
+import {createPackage} from '@shopify/loom';
 
 import {quiltPackage} from '../../config/loom';
 
 export default createPackage((pkg) => {
-  pkg.runtimes(Runtime.Browser, Runtime.Node);
-  pkg.entry({root: './src/index'});
-  pkg.entry({name: 'server', root: './src/server'});
+  pkg.entry({root: './src/index.ts'});
+  pkg.entry({name: 'server', root: './src/server.tsx'});
```

Comment:
Same question as above about the `.tsx` server file

Reply:
[That's the filename in the repo](https://github.com/Shopify/quilt/blob/main/packages/react-effect/src/server.tsx). It's tsx, not ts because it contains JSX content.